provider/baked/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! Tooling for the baked provider.
+
+#[cfg(feature = "export")]
+pub mod export;
+
+pub use icu_provider::prelude::*;
```

Comment:
question:  why do we need this?

also is this crate useful outside of the export mode? it doesn't seem to contain much otherwise, should it be called `baked_exporter` instead?

Reply:
I want to expose zerotrie through it in the next PR.

The prelude export I didn't end up using, but I might change that.

Reply:
Who would depend on this in non exporter contexts?

Reply:
Baked data users, like all our _data crates.

Reply:
I kinda would like those crates to have minimum deps, so they should depend on zerotrie directly and not a re-export from here.

Unless this crate exports something built on top of ZT then I guess that's fine.

But given that these are weird crates that get replaced in places like Google3, having zero deps is really good for them

cc @sffc for thoughts

Reply:
Well we can. Either we keep the policy that baked data needs to be generated with the same minor version, then this is fine, or we commit to stability in 2.0, then this is fine.

Reply:
Ok, `~` deps can resolve this.